packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -21,7 +22,7 @@ general-error-heading = General application error
 basic-error-message = Something went wrong. Please try again later.
 payment-error-1 = Hmm. There was a problem authorizing your payment. Try again or get in touch with your card issuer.
 payment-error-2 = Hmm. There was a problem authorizing your payment. Get in touch with your card issuer.
-payment-error-3 = An unexpected error has occured while processing your payment, please try again.
+payment-error-3b = An unexpected error has occurred while processing your payment, please try again.
```

Comment:
oh boy, these unique ids...

Reply:
yeah. `payment-error-3b` seemed better somehow than `payment-error-3-1` 🤷

Reply:
This is fine, since it's landed, but this didn't need a new ID since it was an english-only typo